src/irmin-pack/unix/s.ml

```diff
@@ -22,11 +22,13 @@ module type S = sig
 
   val integrity_check :
     ?ppf:Format.formatter ->
+    ?heads:commit list ->
```

Comment:
The integrity check -- for stores created with minimal indexing strategy-- needs a reference to a commit. It then traverses that commit. After a snapshot import there is only one commit in the store indeed, but because octez does not use branches, we cannot access it. We need its hash to look it up in index. (There should be only one commit in the index and so we could read it directly from the index I guess). 

For stores that do use branches, the command is indeed somewhat flexible: if no commit is specified the main branch is traversed.  

And also, if used outside of the snapshot import usecase, it can be used to check the integrity of any commit available in the store.